consensus/polybft/commitment.go

```diff
@@ -303,21 +180,21 @@ func (cm *CommitmentMessageSigned) EncodeAbi() ([]byte, error) {
 	}
 
 	data := map[string]interface{}{
-		"bundle":    commitment,
-		"signature": cm.AggSignature.AggregatedSignature,
-		"bitmap":    blsVerificationPart,
+		"commitment": commitment,
```

Comment:
Is there a PR on `core-contracts` for refactoring names on `StateReceiver`? This is still called `bundle` on their side.

Reply:
Asked the SCs team to change it